hands-on-exercise/5-ibc-adv/2-hermes-relayer.md

```diff
@@ -67,7 +67,7 @@ SUBCOMMANDS:
     clear           Clear objects, such as outstanding packets on a channel
     config          Validate Hermes configuration file
     create          Create objects (client, connection, or channel) on chains
-    health-check    Performs a health check of all chains in the the config
+    health-check    Performs a health check of all chains in the config
```

Comment:
@murataniloener Is this not a straight copy of the output?

Reply:
Yes it is

Reply:
```suggestion
    health-check    Performs a health check of all chains in the the config
```

So I'd say we keep it as is, even if it is silly. Thoughts?

Reply:
No I would fix it, perhaps it is already fixed in the most recent version of Hermes.